### PR TITLE
Update: optional fileId in ContentPreview

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #561

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: src/components/ContentPreview/ContentPreview.js [hunk 1]
```diff
@@ -1033,8 +1034,10 @@ class ContentPreview extends PureComponent<Props, State> {
             collection.length > 1 &&
             fileIndex > -1 &&
             fileIndex < collection.length - 1;
-        const fileId = file ? file.id : undefined;
 
+        if (!fileId) {
```
Comment (priyajeet) [inline src/components/ContentPreview/ContentPreview.js]: Will break componentDidMount ContentPreview.js#L244

File: src/components/makeResponsive.js [hunk 2]
```diff
@@ -147,9 +166,14 @@ function makeResponsive(
             );
 
             return (
-                <Measure bounds onResize={this.onResize}>
+                <Measure
+                    bounds
+                    onResize={this.onResize}
+                    innerRef={this.innerRef}
```
Comment (priyajeet) [inline src/components/makeResponsive.js]: Will contentexplorer and picker also need to change?

File: src/components/makeResponsive.js [hunk 3]
```diff
@@ -89,6 +90,24 @@ function makeResponsive(
             this.setState({ size: this.getSize(width) });
         }, 500);
 
+        /**
+         * Callback function for setting the ref which measureRef is attached to
+         *
+         * @return {void}
+         */
+        innerRef = el => {
```
Comment (priyajeet) [inline src/components/makeResponsive.js]: use React.createRef

Comment (DanDeMicco) [reply src/components/makeResponsive.js]: I had to use the callback due to how react-measure is implemented. It has to be a function. Theres an open issue for it https://github.com/souporserious/react-measure/issues/114

**Pull request merged**